android-json-form-wizard/src/main/java/com/vijay/jsonwizard/rules/RulesEngineFactory.java

```diff
@@ -49,7 +52,8 @@ public RulesEngineFactory(Context context, Map<String, String> globalValues) {
         gson = new Gson();
         this.globalValues = globalValues;
         this.rulesEngineHelper = new RulesEngineHelper();
-
+        YamlRuleDefinitionReader yamlRuleDefinitionReader = new YamlRuleDefinitionReader();
+        mvelRuleFactory = new MVELRuleFactory(yamlRuleDefinitionReader);
```

Comment:
Is this change related to the  android-x migration?

Reply:
Its related to `MVELRuleFactory.createRulesFrom(bufferedReader)` being deprecated.